test/common-algorithms/dynamic-programming/sequential/edit_distance.chpl

```diff
@@ -0,0 +1,42 @@
+proc min(x:int, y:int):int{
+	if(x > y){
+		return y;
+	}
+	else{
+		return x;
+	}
+}
+
+proc min(x:int,y:int,z:int):int{
```

Comment:
Use [built-in min](http://chapel.cray.com/docs/master/modules/internal/UtilMisc_forDocs.html#UtilMisc_forDocs.min)